.github/workflows/test-integration-cleanup-template.yaml

```diff
@@ -74,17 +74,7 @@ jobs:
         openshift_server_url: ${{ secrets[matrix.distro.secret.server-url] }}
         openshift_username: ${{ secrets[matrix.distro.secret.username] }}
         openshift_password: ${{ secrets[matrix.distro.secret.password] }}
-    - name: Check Namespace in ${{ matrix.distro.name }} and Delete if Older than 24 hours
+    - name: Delete Namespace on ${{ matrix.distro.name }}
```

Comment:
We also need to set the TTL as an input to the integration template https://github.com/camunda/camunda-platform-helm/blob/b0d805625f669883297ec20af7ceaf92b9ffcff4/.github/workflows/test-integration-template.yaml so the user can define the TTL they need.

Empty TTL label should be deleted immediately (combined with Cleaner `labelFilters` to select the namespaces created with the workflow only).